plugin-flex-ts-template-v2/src/feature-library/canned-responses/config.ts

```diff
@@ -0,0 +1,11 @@
+import { getFeatureFlags } from '../../utils/configuration';
+
+const { enabled = false, location } = getFeatureFlags()?.features?.canned_responses || {};
```

Comment:
Sorry, missed this one earlier- `location` should probably also have a default set here.